services/src/main/java/org/keycloak/exportimport/util/ExportUtils.java

```diff
@@ -106,11 +106,10 @@ public static RealmRepresentation exportRealm(KeycloakSession session, RealmMode
 
         if (options.isClientsIncluded()) {
             clients = realm.getClientsStream().collect(Collectors.toList());
-            List<ClientRepresentation> clientReps = new ArrayList<>();
-            for (ClientModel app : clients) {
-                ClientRepresentation clientRep = exportClient(session, app);
-                clientReps.add(clientRep);
-            }
+            List<ClientRepresentation> clientReps = realm.getClientsStream()
```

Comment:
We are now calling `realm.getClientsStream()` twice, is this on purpose?

Reply:
Thanks, my bad :)